equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/service/CrmFileService.java

```diff
@@ -132,22 +139,24 @@ public JSONObject convertCrmFormLinkedAttachments(JSONObject crmFileJsonObject)
         return linkedAttachments;
     }
 
-    public void convertCrmFormChargesBrought(JSONObject crmFileJsonObject) {
-        JSONObject chargesBrought;
-        if (!crmFileJsonObject.has(CRM14_CHARGES_BROUGHT)) {
-            chargesBrought = new JSONObject();
-            chargesBrought.put(CRM14_CHARGES_BROUGHT, new JSONArray());
+    public void convertCrmFormObjectToArray(JSONObject crmFileJsonObject, String jsonParentKey, String jsonChildKey) {
+        JSONObject jsonParentObject;
+        if (!crmFileJsonObject.has(jsonParentKey)) {
+            jsonParentObject = new JSONObject();
+            jsonParentObject.put(jsonParentKey, new JSONArray());
+            return;
         }
-        chargesBrought = (JSONObject) crmFileJsonObject.get(CRM14_CHARGES_BROUGHT);
+        jsonParentObject = (JSONObject) crmFileJsonObject.get(jsonParentKey);
 
-        if (!chargesBrought.has(CRM14_CHARGES)) {
-            chargesBrought.put(CRM14_CHARGES, new JSONArray());
+        if (!jsonParentObject.has(jsonChildKey)) {
+            jsonParentObject.put(jsonChildKey, new JSONArray());
+            return;
         }
-        if (chargesBrought.get(CRM14_CHARGES) instanceof JSONObject) {
-            log.warn("CRM14 eForm Charges Brought to be a list. Try converting into a list");
+        if (jsonParentObject.get(jsonChildKey) instanceof JSONObject) {
+            log.warn("CRM14&15 eForm "+ jsonParentKey + " Charges Brought to be a list. Try converting into a list");
```

Comment:
this is a generic function to convert any object into array. Log message should be as that and not specific to either CRM14/15 nor charges brought.
Also, I am personally not a fan of conversion function to also alter the source. I prefer the conversion function to simply return the converted structured and let the caller decide what to do with this. 
With this version, we have 2 "convert" functions with different behaviours (see: convertCrmFormLinkedAttachments)
However, that should also be using the generic new version.